Home.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type="text/css" href="Styles.css">
+    <link rel="stylesheet" type ="text/css" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio | Home</title>
+</head>
+
+<body>
+    <header>
+        <a href="Home.html" class="Homebutton">
+            James
+        </a>
+        <nav class="topnav">
+            <a href="#">
+                About
+            </a>
+            <a href="#">
+                Portfolio
+            </a>
+            <a href="#">
+                Contact Me
+            </a>
+        </nav>
+        <nav class="sociallinks">
+            <a href="https://github.com/James-Benson">
+                <i class="fab fa-github"></i>
+            </a>
+            <a href="#">
+                <i class="fab fa-facebook-square"></i>
+            </a>
+            <a href="#">
+                <i class="fab fa-twitter-square"></i>
+            </a>
+        </nav>
+    </header>
+
+    <main>
+        <span>THERE'S GOING TO BE SOME CONTENT HERE LATER BUT I JUST NEED TO FILL LOTS OF SPACE AT THE MOMENT TO TEST</span>
```

Comment:
I feel like if you increased the contrast between the text color and the bright starburst of the background the text would be slightly easier to read at the wider resolutions. At thinner resolutions the text is more spread out over the black abyss of space and is therefore slightly easier to read. This is entirely nitpicky, feel free to leave in you want.